03-attribute-operations.Rmd

```diff
@@ -405,13 +405,13 @@ These aggregating functions return `sf` objects with rows representing continent
 \index{attribute!subsetting}
 \index{attribute!aggregation}
 Let's combine what we have learned so far about **dplyr** functions, by chaining multiple commands to summarize attribute data about countries worldwide by continent.
-The following command calculates population density (with `mutate()`), arranges continents by the number countries they contain (with `arrange()`), and keeps only the 3 most populous continents (with `slice_max()`), the result of which is presented in Table \@ref(tab:continents)):
+The following command calculates population density (with `mutate()`), arranges continents by the number of countries they contain (with `arrange()`), and keeps only the three most populous continents (with `slice_max()`), the result of which is presented in Table \@ref(tab:continents)):
 
 ```{r 03-attribute-operations-30}
 world_agg5 = world |> 
   st_drop_geometry() |>                      # drop the geometry for speed
   select(pop, continent, area_km2) |> # subset the columns of interest  
-  group_by(continent) |>                     # group by continent and summarize:
+  group_by(Continent = continent) |>  # group by continent and summarize:
```

Comment:
@Robinlovelace would you write your own code like this?

Reply:
Yes, if the aim was to have "Continent" consistently capitalised in the resulting table. How would you write it @Nowosad?